applications/matter_bridge/sysbuild/ipc_radio/nrf54h20dk_nrf54h20_cpurad.conf

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
```

Comment:
We don't need these two files in the ipc_radio directory, as we don't use this image.